lib/Embargo/Embargo.js

```diff
@@ -17,13 +18,13 @@ export default class Embargo extends React.Component {
       })
     }
 
```

Comment:
Add a `defaultProp` here 

```suggestion
      static defaultProps = {
           alignment: 'left'
      };
```

Reply:
If we solve it like this, why can't the default alignment be center? Because it's the default now, we don't have to pass something to have the Embargo line centered. Couldn't we just leave it like this?

Reply:
That make sense